hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/webapp/TestHsWebServicesJobsQuery.java

```diff
@@ -136,7 +130,8 @@ public void testJobsQueryStateNone() throws JSONException, Exception {
     assertEquals(MediaType.APPLICATION_JSON_TYPE, response.getType());
     JSONObject json = response.getEntity(JSONObject.class);
     assertEquals("incorrect number of elements", 1, json.length());
-    assertEquals("jobs is not null", JSONObject.NULL, json.get("jobs"));
+    assertEquals("jobs is not empty",
```

Comment:
I found one incompatible change about Jersey - after upgrading from 1.12 to 1.13, the root element whose content is empty collection is changed from null to empty object({}). Related to following change: https://java.net/jira/browse/JERSEY-1168

This change of lines fixes tests and assertions about JSONObject.NULL to address the change above.